### PR TITLE
Documentation Update

### PR DESCRIPTION
**Pull request by jgmelber**
This adds a simplified setup process for Linux. I am considering adding this to the main README rather than a separate doc in a subdirectory

### CODE REVIEW

File: docs/quickSetupLin.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+   ```bash
+   sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+   sudo apt update
+   sudo apt install gcc-13 g++-13 -y
+   sudo apt install libopencv-dev python3-opencv
+   ```
+
+1. Remember to source the Vitis™ AIE Essentials setup script from [above](#install-aietools).
+   
+1. Remeber to source the xrt setup script: `source /opt/xilinx/xrt/setup.sh`
```
Comment (AndraBisca) [inline docs/quickSetupLin.md:157]: Typo on 'remember'

File: docs/quickSetupLin.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+> Remember to set up your environment including Vitis™ AIE Essentials, your license, XRT, and IRON
+> ```
+>   source yourVitisSetupScript.sh
+>   export LM_LICENSE_FILE=/opt/Xilinx.lic
+>   source /opt/xilinx/xrt/setup.sh
+>   source utils/env_setup.sh my_install/mlir_aie my_install/mlir my_install/llvm-aie
+> ```
+
+For your design of interest, for instance from [programming_examples](../programming_examples/), 2 steps are needed: (i) build the AIE desgin and then (ii) build the host code.
```
Comment (AndraBisca) [inline docs/quickSetupLin.md:180]: Typo on 'design'

**Review by AndraBisca — APPROVED** [review]

**Comment by AndraBisca** [comment]
I think it would be good to have all build instructions easily available at the top README, or at least a link there pointing to the instructions in a subdirectory (if the README gets too lengthy for example).

**Comment by jgmelber** [comment]
> I think it would be good to have all build instructions easily available at the main README, or at least a link there pointing to the instructions in a subdirectory (if the README gets too lengthy for example).

We have links now but I want to put something on the main page for those interested in getting the programming examples/guide running quickly

File: docs/quickSetupLin.md (hunk 1 above)
Comment (jgmelber) [inline docs/quickSetupLin.md:157]:
```suggestion
1. Remember to source the XRT setup script: `source /opt/xilinx/xrt/setup.sh`
```

File: docs/quickSetupLin.md (hunk 2 above)
Comment (jgmelber) [inline docs/quickSetupLin.md:180]:
```suggestion
For your design of interest, for instance from [programming_examples](../programming_examples/), 2 steps are needed: (i) build the AIE design and then (ii) build the host code.
```

**Comment by jgmelber** [comment]
Needs #1852

File: README.md [hunk 3]
```diff
@@ -16,6 +16,211 @@ This repository contains an [MLIR-based](https://mlir.llvm.org/) toolchain for A
...
+    1. Extract the required tools:
+
+       ``` bash
+          tar -xzvf ryzen_ai-1.3.0ea1.tgz
+          cd ryzen_ai-1.3.0
+          mkdir vitis_aie_essentials
+          mv vitis_aie_essentials*.whl vitis_aie_essentials
+          cd vitis_aie_essentials
+          unzip vitis_aie_essentials*.whl
+       ```
```
Comment (fifield) [inline README.md:58]:
```suggestion
    1. Download VAIML Installer for Linux based compilation: `ryzen_ai-1.3.0ea1.tgz`
 
    1. Install the tools:

       ``` bash
          tar -xzvf ryzen_ai-1.3.0ea1.tgz
          cd ryzen_ai-1.3.0
          ./install_ryzen_ai_1_3.sh
       ```
```
mlir-aie [supports the installer it comes with](https://github.com/Xilinx/mlir-aie/actions/workflows/buildAndTestRyzenAISw.yml).

Comment (fifield) [reply README.md:58]: The downside of installing it via the installer or via `pip install vitis_aie_essentials` is that it pins you to python 3.10

**Review by fifield — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (jgmelber) [reply README.md:58]: For that reason I will stick with unzipping. I will investigate a workaround for pinning the Python version later

**Comment by jgmelber** [comment]
#1859 fixes the quick setup script to support Python 3.12 in Ubuntu 24.10

**Pull request merged**